Transbank/WebpayRest/Oneclick/Requests/MallFinishRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net.Http;
+using Transbank.Common;
+
+namespace Transbank.Webpay.Oneclick.Requests
+{
+    internal class MallFinishRequest : BaseRequest
+    {
+        internal MallFinishRequest(string token)
+            : base($"/rswebpaytransaction/api/oneclick/v1.0/inscriptions/{token}",
+                  HttpMethod.Put){}
+    }
+}
```

Comment:
What is the difference between this `MallFinishRequest` class and `FinishRequest` if the only Oneclick available is Mall?

Reply:
This was done to retain consistency with the others products. We discussed it on Slack.